docs/_data/integrations/integrations.yml

```diff
@@ -0,0 +1,1200 @@
+- desc: Interacts with Docker to provide cloud-native capabilities for
```

Comment:
@Boombag0607 good work. How is the update of this set of integration information programmatically updated? We need this to be fed from the Meshery Integrations spreadsheet. Look to the `/scripts` and see that this information is pulled and pushed to layer5.io and meshery.io currently. Please open a new issue to have that information pushed to docs.meshery.io (to here), too.

Reply:
Icons (colored SVGs) and primary component color should be included.

Reply:
The descriptions do not update right now. I'll open the issue so that the changes in the spreadsheet get pushed here. I can't access the spreadsheet, so correct me if I am wrong but is [this](https://github.com/meshery/meshery.io/blob/master/integrations/data.json) the data pushed currently?